synapse/handlers/federation.py

```diff
@@ -80,6 +81,24 @@
 logger = logging.getLogger(__name__)
 
 
+backfill_processing_before_timer = Histogram(
+    "synapse_federation_backfill_processing_before_time_seconds",
+    "sec",
+    [],
+    buckets=(
+        1.0,
+        5.0,
+        10.0,
+        20.0,
+        30.0,
+        40.0,
+        60.0,
+        80.0,
+        "+Inf",
+    ),
```

Comment:
We will probably want to adjust the fidelity in the buckets once we have some real data.

I've seen the linearizer lock take 45s and calculating the `likely_domains` take 20s for example